README.md

```diff
@@ -85,6 +88,165 @@ public class TokenGenerationUtil {
+
+[Example](https://github.com/skyflowapi/skyflow-java/blob/master/samples/src/main/java/com/example/BearerTokenWithContextGeneration.java):
+
+``` java
+import com.skyflow.entities.ResponseToken;
+import com.skyflow.errors.SkyflowException;
+import com.skyflow.serviceaccount.util.Token;
+import java.io.File;
+
+public class BearerTokenWithContextGeneration {
+    public static void main(String args[]) {
+
+        String bearerToken = null;
+
+        // Generate BearerToken with context by specifying credentials.json file path
+        try {
+            String filePath = "<YOUR_CREDENTIALS_FILE_PATH>";
+            BearerToken token = new BearerToken.BearerTokenBuilder()
+                    .setCredentials(new File(filePath))
+                    .setContext("abc")
+                    .build();
+
+            bearerToken = token.getBearerToken();
+            System.out.println(bearerToken);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        // Generate BearerToken with context by specifying credentials.json as string
+        try {
+            String fileContents = "<YOUR_CREDENTIALS_FILE_CONTENTS_AS_STRING>";
+            BearerToken token = new BearerToken.BearerTokenBuilder()
+                    .setCredentials(fileContents)
+                    .setContext("abc")
+                    .build();
+
+            bearerToken = token.getBearerToken();
+            System.out.println(bearerToken);
+
+        } catch (SkyflowException e) {
+            e.printStackTrace();
+        }
+
+    }
+}
+
+```
+
+Note: 
+- Either credentials file path or credentials as string can be passed to `setCredentials` method of BearerTokenBuilder class.
+- If both credentials file path and credentials string are passed to `setCredentials` method, then setter which is at the last will be taken precedence over the other.
+
+## Service Account Scoped Bearer Token Generation
+
+A service account having multiple roles can be used to generate bearer tokens with restricted access by providing the appropriate `roleID`. The token generated is valid for 60 minutes and perform opertaions with permissions associated with the role.
+
+[Example](https://github.com/skyflowapi/skyflow-java/blob/master/samples/src/main/java/com/example/ScopedTokenGeneration.java):
+
+```java
+import java.io.File;
+
+public class ScopedTokenGeneration {
+    public static void main(String args[]) {
+
+        String scopedToken = null;
+
+        // Generate Scoped Token  by specifying credentials.json file path
+        try {
+            String filePath = "<YOUR_CREDENTIALS_FILE_PATH>";
+            BearerToken token = new BearerToken.BearerTokenBuilder()
```

Comment:
Make sure all code is run through a linter. The spacing here looks off from the standard.

Reply:
Done